cloud-functions/annotation-function/main.py

```diff
@@ -1,19 +1,54 @@
 import os
 import youtube_dl
+import cv2
 from google.cloud import storage
 from google.cloud import pubsub_v1
+from google.cloud import videointelligence
+import io
+import base64
 
 import functions_framework
 
 @functions_framework.http
 def handle(request):
     # This function should call the Google Vision/Video API and return
-    
+    """handle a request to the function"""
```

Comment:
Update the terraform for this function to increase `timeout`

Reply:
Added another commit to address this. Timeout is now set dynamically to allow time to capture and analyse the video.